src/promnesia/server.py

```diff
@@ -131,14 +133,24 @@ def search_common(url: str, where):
     url = canonify(url)
     logger.info('normalised url: %s', url)
 
+    result = {
+        'orginal_url'   : original_url,
+        'normalised_url': url,
+        'visits': []
+    }
+
     engine, binder, table = get_stuff()
 
     query = table.select().where(where(table=table, url=url))
 
     logger.info('query: %s', query)
 
     with engine.connect() as conn:
-        visits = [binder.from_row(row) for row in conn.execute(query)]
+        try:
+            visits = [binder.from_row(row) for row in conn.execute(query)]
+        except sqlalchemy.exc.OperationalError as e:
+            if e.msg == 'no such table: visits':
+                return result
```

Comment:
Hm, that would be misleading, e.g. when you have some error, you'd get zero visits in the backend. Wondering what would be your usecase?
One thing I can think of is when you're using the extension and the backend isn't working, you don't get 'local' visits either. I guess the right way to handle this would be to support displaying both errors and valid results at the same time in sidebar.

Reply:
hmm, displaying results + errors/other messages would definitely be a move in the right direction. I'll see if the codebase can handle that without too much hassle. 
My usecase was very dubious - eliminating exception traces from server log. See, server starts up, but until indexer first runs, you're getting these exceptions that the database doesn't exist. Took me a while to conclude that server doesn't have any "init empty" db code, that it's just waiting on the indexer to do it.